src/ntfsck.c

```diff
@@ -1162,6 +1162,49 @@ static int ntfsck_add_index_entry_orphaned_file(ntfs_volume *vol, s64 mft_no)
 	goto next_inode;
 }
 
+static void ntfsck_check_mft_record_unused(ntfs_volume *vol, s64 mft_num)
+{
+	MFT_RECORD *mrec;
+	u16 seq_no;
+	s64 pos = mft_num * vol->mft_record_size;
```

Comment:
is it right calculation of pos variable ??
position of number 0 mft record is zero?

Reply:
mft_num will not be zero. mft_num will start to be FILE_first_user(16).